docs/fds-components/vendors~main.b1b30b22f6e5ff8a72c3.bundle.js.LICENSE.txt

```diff
@@ -106,7 +83,7 @@ object-assign
  */
 
 /** @license React v16.13.1
- * react-is.development.js
+ * react-is.production.min.js
```

Comment:
what the......

Reply:
Wow that's weird, looks like it changed in Steven's last PR: https://github.com/cbinsights/form-design-system/pull/658/files